lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -606,10 +606,16 @@ createToLayoutOp(PatternRewriter &rewriter, Location loc, Value input,
   auto desiredLayout =
       rewriter.getAttr<LayoutAttr>(ty, desiredMemorySpace, currLayout.getGrid(),
                                    desiredElementType, desiredMemLayout);
+  tensor::EmptyOp exising_empty = input.getDefiningOp<tensor::EmptyOp>();
+  // If we are changing the layout of existing op
+  // we need to set the insertion point to the existing op
+  if (exising_empty) {
+    rewriter.setInsertionPoint(input.getDefiningOp());
+  }
```

Comment:
So I'm thinking now that this `setInsertionPoint` might not be necessary anymore.  The insertion point will have already been set to be the current op we're rewriting so things should "just work".